app/app.go

```diff
@@ -993,7 +1078,13 @@ func (app *OraichainApp) upgradeHandler() {
 
 	if upgradeInfo.Name == BinaryVersion && !app.upgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
 		storeUpgrades := storetypes.StoreUpgrades{
-			Added: []string{icacontrollertypes.SubModuleName, icahosttypes.SubModuleName, ibcfeetypes.ModuleName, intertxtypes.ModuleName},
+			Added: []string{
+				icacontrollertypes.SubModuleName,
+				icahosttypes.SubModuleName,
+				ibcfeetypes.ModuleName,
+				intertxtypes.ModuleName,
+				providertypes.ModuleName,
```

Comment:
We only need to add providertypes. Other modules have already been added in the latest version

Reply:
I just add providertypes to upgrade module

Reply:
Yea but we dont need to add other modules though. There will be an error if we re-add them since they have already been added in the version v0.41.2